src/runtime_src/core/pcie/emulation/cpu_em/generic_pcie_hal2/shim.h

```diff
@@ -494,6 +542,7 @@ namespace xclcpuemhal2 {
       exec_core* mCore;
       SWScheduler* mSWSch;
       bool mIsKdsSwEmu;
+      std::atomic_bool mDeviceProcess;
```

Comment:
Why?
Are the process stored in 1 bit in your OS?
The names in a program have a meaning, this is not random characters.